Sources/SocialAuthentication/AppleAuthentication/AppleAuthentication.swift

```diff
@@ -71,10 +66,89 @@ extension AppleAuthentication {
     }
 }
 
-// MARK: - UserId
+// MARK: - Sign Out
+
+extension AppleAuthentication {
+    
+    public func signOut() {
+        
+        deletePersistedUser()
+    }
+}
+
+// MARK: - User
+
+extension AppleAuthentication {
+    
+    public func getCurrentUserProfile() -> AppleUserProfile? {
+        
+        guard
```

Comment:
Hey @rachaelblue with using the guard here if any one of these values are null we won't get any profile data back. Any of these values could be null in ASAuthorizationAppleIDCredential. https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidcredential

Reply:
Changed the `AppleUserProfile` attributes to optionals.